tests/countries/test_sint_maarten.py

```
@@ -0,0 +1,252 @@
+            "2023-05-28",
+            "2024-05-19",
+            "2025-06-08",
+        )
+        self.assertHolidayName(name, range(2011, 2050))
+
+    def test_emancipation_day(self):
+        name = "Emancipation Day"
+        self.assertNoHolidayName(name, range(1900, 2012))
+        self.assertHolidayName(name, (f"{year}-07-01" for year in range(2012, 2050)))
+
+    def test_constitution_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)